WordPressAuthenticator/Authenticator/WordPressAuthenticatorDelegateProtocol.swift

```diff
@@ -68,6 +68,14 @@ public protocol WordPressAuthenticatorDelegate: class {
     ///
     func shouldPresentLoginEpilogue(isJetpackLogin: Bool) -> Bool
 
+    /// Indicates the Host app wants to handle and display a given error.
```

Comment:
These two methods would be new. This would WordPress, but the fix is simple: for host apps that are good with WPAuthenticator's error handling, shoudlDisplayError would just return false, and handleError would do nothing

Reply:
Thanks for the info! After this is merged, I'll create a WP PR to address this.